website/src/app/kb/authenticate/okta/readme.mdx

```diff
@@ -0,0 +1,262 @@
+import Alert from "@/components/DocsAlert";
+import PlanBadge from "@/components/PlanBadge";
+import Image from "next/image";
+import Link from "next/link";
+
+<PlanBadge plans={["enterprise"]}>
+
+# SSO with Okta
+
+</PlanBadge>
+
+Firezone integrates with Okta using a custom connector that supports both
+authentication and directory sync. Use this guide if you're looking to setup SSO
+with Okta for your Firezone Enterprise account and want to automatically sync
+users and groups from Okta to Firezone.
+
+<Alert color="info">
+  If you're just looking to authenticate users against Okta **without**
+  automatic directory sync, use our [universal OIDC
+  connector](/kb/authenticate/oidc) instead, available on all plans.
+</Alert>
+
+## Overview
+
+The Firezone Okta connector integrates with Okta's APIs to support user
+authentication and directory sync.
+
+Users, groups, and organizational units are synced every few minutes to ensure
```

Comment:
Was this copied from the Google Workspace page?  I don't think Okta has organizational units does it?